frontend/README.md

```diff
@@ -1,5 +1,13 @@
 # Project created with React + TypeScript + Vite. Uses JSON Server to simulate an external Data Base
```

Comment:
Formating for main heading = ## and could u update this as well, JSON server is not used, and I think the title could be more precise.

Reply:
Thank you, I missed the formatting and the JSON server part.